virttest/libvirt_xml/devices/tpm.py

```diff
@@ -92,11 +92,18 @@ class ActivePCRBanks(base.base.LibvirtXMLBase):
             Tpm active_pcr_banks xml class.
 
             Elements:
-            pcrbank_list: list of elements
+            sha1: supported element sha1
+            sha256: supported element sha256
+            sha384: supported element sha384
+            sha512: supported element sha512
+            pcrbank_list: list of any customized elements
```

Comment:
Could you please add a comment or 'FIXME' to this line to explain that it does not support setup_attrs()?
So others can get information quickly when facing the same issue.

Reply:
Done. Thanks.
-            pcrbank_list: list of any customized elements
+            pcrbank_list: list of any customized elements. It does not support setup_attrs().